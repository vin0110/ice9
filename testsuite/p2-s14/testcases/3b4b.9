var i: int[5];
i[0] := 0;
fa a := 0 to 5 ->
    write i[a];
    fa a := 0 to i ->
        write a;
    af 
af
