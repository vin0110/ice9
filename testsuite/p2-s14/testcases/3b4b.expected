line 5: Semantic error: Bounds must be ints
